server/shema/resolvers/mutationResolvers.js

```diff
@@ -0,0 +1,95 @@
+const Users = require("../../models/UsersModels");
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcryptjs");
+const { Op } = require("sequelize");
+const { adminFeatures, userFeatures } = require("../../features/features");
+const { EDIT, CREATE, DELETE } = require("../../features/featureConstants");
+const logger = require("../../logger/config");
+
+const mutations = {
+  async setUserInfo(parent, { key, value }, { redis }) {
+    try {
+      await redis.set(key, JSON.stringify(value));
+      return true;
+    } catch (e) {
+      logger.error(`Failed to set user in redis: ${e.message}`);
+      return false;
+    }
+  },
+
+  async createUser(root, { data }, context) {
+    if (context.currentUser && context.currentUser.features.includes(CREATE)) {
```

Comment:
I would be nice to separate authentication logic from data persistence layer logic.
Also, seems like we can generalize this somehow